infra/terraform/modules/data_access/common/sentry/sentry/sentry.py

```diff
@@ -0,0 +1,26 @@
+'''
+Environment variables:
+ - ENV, e.g. "dev", "alpha", etc...
+ - SENTRY_DSN, Sentry DSN
+'''
+
+import os
+
+from raven import Client as Sentry
+from raven.transport.http import HTTPTransport as SentryHTTPTransport
+
+
+def catch_exceptions(fn):
+    def wrapped(*args, **kwargs):
+        try:
+            fn(*args, **kwargs)
+        except Exception:
+            client = Sentry(
+                os.environ["SENTRY_DSN"],
+                transport=SentryHTTPTransport,
+                environment=os.environ["ENV"],
+            )
+            client.captureException()
+            raise
+
+    return wrapped
```

Comment:
This path `common/sentry/sentry/sentry.py` seems like a code smell to me. Is there any reason we can't have the `sentry` module and the `naming` module in the same package with a single `setup.py`? We could call it `moj_analytics_tools` (or something) instead of `common` and then `from moj_analytics_tools import sentry` or `from moj_analytics_tools.sentry import catch_exceptions`. I don't think the size of the combined modules would be too big to upload with the lambda functions.